app/lib/submission_builder/ty2022/states/ny/documents/rtn_header.rb

```diff
@@ -30,12 +36,13 @@ def document
                 unless @submission.data_source.date_electronic_withdrawal.nil?
                   xml.ELC_AUTH_EFCTV_DT claimed: @submission.data_source.date_electronic_withdrawal
                 end
-                # xml.PYMT_AMT
-                # xml.DCMT_RCVD_DT
-                # xml.PSTMRK_DT
-                xml.ACH_IND
-                xml.RFND_OWE_IND
+                unless @submission.data_source.withdraw_amount.nil?
+                  xml.PYMT_AMT claimed: @submission.data_source.withdraw_amount
+                end
+                xml.ACH_IND claimed: @submission.data_source.ach_debit_transaction? ? 1 : 2
+                xml.RFND_OWE_IND claimed:  REFUND_OR_OWE_TYPES[@submission.data_source.refund_or_owe_taxes_type]
```

Comment:
There's an extra space here
```suggestion
                xml.RFND_OWE_IND claimed: REFUND_OR_OWE_TYPES[@submission.data_source.refund_or_owe_taxes_type]
```